### ISSUE TITLE
↔ [Converter] Add support for layer_norm in TRTorch

### ISSUE THREAD

**Issue by narendasan** [NVIDIA/TRTorch#278]
## aten::layer_norm

- **Function Schema**:

`aten::layer_norm(Tensor input, int[]   normalized_shape, Tensor? weight=None, Tensor? bias=None, float   eps=1.0e-05, bool cudnn_enable=True) -> (Tensor)`

- **Original PyTorch API**:

https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html#torch.nn.LayerNorm

- **Relevant TensorRT Documentation**:

**Comment by xylcbd** [comment]
upgrade for ln is request: https://github.com/pytorch/pytorch/blob/v1.13.0-rc1/torch/onnx/symbolic_opset17.py
